extra/import_data.py

```diff
@@ -0,0 +1,174 @@
+            return
+        elif last_updated_on_Alissa > last_updated_on_mongoDB:
+            db.delete_many({"dn_no": patient_dn_no})
+            logger.info('removed: %s from database, start replacing with newer version' % patient_dn_no)
+        elif last_updated_on_Alissa < last_updated_on_mongoDB:
+            logger.info('lastUpdatedOn older than within the database for %s, this should not happen' % patient_dn_no)
+            # TODO send (email) notification of this error, see issue #11 GitHub
+
+    # get all relevant info from one patient into one dictionary.
+    patient = {}
+    patient["dn_no"] = patient_dn_no
+    logger.info('start uploading to mongodb: %s' % patient_dn_no)
+    patient["patient_accession_no"] = accession_number
+    patient.update(inheritance_analysis)
+    patient["analysis"] = patient.pop("reference")
+    patient["platform_dataset"] = analyis_sources["platformDataSet"]["info"]
+
+    # changes the format of the annotation sources from Alissa, to fit in MongoDB and resemble the old O-schijf format
+    externalSources_dict = {}
+    for source in analyis_sources['annotationSources']:
+        source_name = ""
+        source_value = ""
+        for key, value in source.items():
+            if key == 'name':
+                source_name = value
+            elif key == 'description':
+                source_value = value
+        if source_name and source_value:
+            externalSources_dict[source_name] = source_value
+        else:
+            logger.info("error, externalsources has changed format in Alissa")
+    patient["annotation_sources"] = externalSources_dict  # add previous section to total patient info
+
+    # extract information from the VUS/variant data and add to "patient"
+    for variant in vus_export:
+        # format for gnomad Links. 4 availble in Alissa (snp, insertion, deletion and substitution)
+        try:
+            fullgnomen = variant['platformDatasets']['HGVS genomic-level nomenclature (fullGNomen)']
+            variant['fullgnomen'] = fullgnomen  # NC_000001.10:g.12345678T>A
+            if fullgnomen:  # NC_000001.10:g.12345678T>A
+                gnomad_data = re.split(':[a-z].', fullgnomen)[1]  # 123456789T>A
+                if variant["type"] == "snp":
+                    gnomad_data = [c for c in re.split(r'([-+]?\d*\.\d+|\d+)', gnomad_data) if c]  # 123456789T>A
+                    gnomad_data = variant["chromosome"] + "-" + re.sub('[<>]+', '-', ("-".join(gnomad_data)))  # 1-12345678-T-A
+                    variant['GnomadVariant'] = {'Single nucleotide variant': gnomad_data}
+                elif variant["type"] in ["insertion", "deletion", "substitution"]:
+                    gnomad_data = variant["chromosome"] + "-" + gnomad_data
+                    variant['GnomadVariant'] = {variant["type"].capitalize(): gnomad_data}
+                    # TODO: make link format correctly for "insertion", "deletion", "substitution" genomic variations
```

Comment:
In my opinion, those kind of TODO comments should not be part of a v1.0.0 release.

Reply:
see earlier comments